canonical-web-and-design/juju.is
# About architecture

## QA
- Go to http://0.0.0.0:8041/about/architecture
- Check the content matches the [copy doc](https://docs.google.com/document/d/1DfRx28l_xePg2Jx-0VsJc7BM5Oez4L4l_UEdQxQTG1o/edit#)

Fixes https://github.com/canonical-web-and-design/juju.is/issues/105
